Components/GridPostsSlider/script.js

```diff
@@ -36,21 +36,22 @@ class GridPostsSlider extends window.HTMLDivElement {
     const { options } = this.props
     const config = {
       a11y: options.a11y,
-      spaceBetween: 23,
+      spaceBetween: 24, // $gutter-width
       pagination: {
         el: this.$pagination,
         clickable: true
       },
-      slidesPerView: 4,
+      slidesPerView: 1,
+      breakpointsInverse: true,
       breakpoints: {
         480: {
-          slidesPerView: 1
-        },
-        768: {
           slidesPerView: 2
         },
-        1024: {
+        768: {
           slidesPerView: 3
+        },
+        1280: {
+          slidesPerView: 4
```

Comment:
should we add another one for super large screens? 6 items from 1440?

Reply:
That is a good point. I've decided to go with 5 cards at 1440px screen width, instead of 6, because the they look a bit cluttered at that particular break 
![ssliider](https://user-images.githubusercontent.com/49150551/61933088-48e52a00-af85-11e9-8437-4013272ad90f.png)
point. I did, however, add a layout with 6 cards at 1920px screen width.